docs/node/workflow-apis.md

```diff
@@ -55,6 +55,27 @@ If you wanted to send data in, you probably will want to read data out.
 <!--SNIPSTART nodejs-blocked-workflow-->
 <!--SNIPEND-->
 
+<details>
+  <summary>
+    Why <em>NOT</em> <code>new Signal</code> and <code>new Query</code>?
+  </summary>
+
+The semantic of `defineSignal`/`defineQuery` is intentional, in that they return Signal/Query **Definitions**, not unique instances of Signals and Queries themselves.
```

Comment:
roey comments:

- See more info in my summary in the proposal that may be relevant to add here: https://github.com/temporalio/proposals/pull/44#issuecomment-943347218
- This too: https://temporalio.slack.com/archives/C01DKSMU94L/p1634287094208800
-  I'd add this: https://github.com/temporalio/sdk-node/pull/301#discussion_r729161584